package.json

```diff
@@ -8,9 +8,10 @@
     "bootstrap": "lerna bootstrap",
     "build": "lerna run build",
     "lint": "eslint './**/**/*.ts'",
-    "lint-fix": "eslint './**/**/*.ts' --fix"
+    "lint-fix": "eslint './**/**/*.ts' --fix",
+    "test": "lerna run test --ignore maci-common --ignore maci-server --ignore maci-integrationtests --ignore maci-cli"
```

Comment:
Why ignore `maci-cli`? Is that because you'll plan to refactor that as part of #794?

Reply:
I also wonder how intuitive it is to ignore integration tests 🤔 but I think it makes sense

Reply:
Basically here I'm trying to run only in unit tests, hence why cli and integration tests are ignored

Reply:
> Running them in this command would involve first spawning a local hardhat node

But doesn't running contract tests also spin up a hardhat node?

Reply:
It does, however because we run them using hardhat directly (hardhat test x.test.ts) there is no need to pre spawn a node, it just does it automatically and kills it once done